.pre-commit-config.yaml

```diff
@@ -19,7 +20,9 @@ repos:
     hooks:
       - id: hadolint
         args:
+          - --failure-threshold=warning
           - --ignore=DL3008
+          - --ignore=DL3007
```

Comment:
yeah I think this makes sense here - the base image only has one tag anyway and that's `latest`